src/main/kotlin/no/nav/familie/ba/sak/kjerne/steg/BehandlingSteg.kt

```diff
@@ -327,7 +342,7 @@ fun hentNesteSteg(behandling: Behandling, utførendeStegType: StegType): StegTyp
             when (utførendeStegType) {
                 REGISTRERE_PERSONGRUNNLAG -> VILKÅRSVURDERING
                 VILKÅRSVURDERING -> BEHANDLINGSRESULTAT
-                BEHANDLINGSRESULTAT -> if (behandling.resultat == Behandlingsresultat.ENDRET_UTBETALING) {
+                BEHANDLINGSRESULTAT -> if (endringerIUtbetaling == EndringerIUtbetaling.ENDRING_I_UTBETALING) {
                     IVERKSETT_MOT_OPPDRAG
                 } else if (behandling.kategori == BehandlingKategori.EØS && behandling.resultat == Behandlingsresultat.FORTSATT_INNVILGET) {
                     FERDIGSTILLE_BEHANDLING
```

Comment:
Denne vil jeg gjerne undersøke nærmere! Hadde vært fint å skrive seg helt bort fra å bruke behandlingsresultat i stegmotoren. + feilmeldingen på linjen under er litt rar å kaste inne i stegfilen 🤔

Reply:
Godt poeng! 

Jeg lurer på om man kan slå til med å endre det til 
```suggestion
                } else if (behandling.kategori == BehandlingKategori.EØS && endringerIUtbetaling == EndringerIUtbetaling.INGEN_ENDRING_I_UTBETALING) {
                    FERDIGSTILLE_BEHANDLING
```

Har ikke satt meg inn i hvordan det er med EØS behandlinger, men kan ta en sjekk med Eivind 😊